File: handler.py
import json

def process_sns(event, context):
    print("+++++++++++++++++++++++++++++++++++++++")
    print(event)
    # for record in event['Records']:
    #     sns_message = record['Sns']
    #     print(f"Mensaje recibido de SNS: {sns_message['Message']}")
    #     # Procesa el mensaje según sea necesario
    #     print(f"Detalles del mensaje completo: {json.dumps(sns_message, indent=2)}")

    # return {
    #     "statusCode": 200,
    #     "body": json.dumps({"message": "Mensajes procesados correctamente"})
    # }

File: handler.js
module.exports.process_sns = (event, context) => {
    console.log("**Ingreso de NODEJS process_sns**")
    console.log("event::"+JSON.stringify(event));

    const response = {
        statusCode: 200,
        body: JSON.stringify({
          message: `Mensaje procesado`,
        }),
      };
  
      return response; // Retornamos el objeto que AWS usará como respuesta
};